Modules/_ssl/misc.c

```diff
@@ -15,7 +15,14 @@ _PySSL_BytesFromBIO(_sslmodulestate *state, BIO *bio)
         PyErr_SetString(PyExc_ValueError, "Not a memory BIO");
         return NULL;
     }
-    return PyBytes_FromStringAndSize(data, size);
+
+    PyBytesWriter *writer = PyBytesWriter_Create(size);
+    if (writer == NULL) {
+        return NULL;
+    }
+    char *str = PyBytesWriter_GetData(writer);
+    memcpy(str, data, size);
```

Comment:
Hmm, @vstinner,
Isn’t PyBytesWriter_WriteBytes not really intended for this use case? Looking at the current codebase, most cases seem to just use memcpy.